src/ResourceManager/DataLakeStore/Commands.DataLakeStore/ChangeLog.md

```diff
@@ -18,7 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+* Update the sdk version of dataplane to 1.1.14 to include SDK fixes
```

Comment:
This is meant to be customer-facing information,  please uodate this  with customer-facing changes that customers will understand

Reply:
@markcowl I have added the link to our SDKs changelog. That should do right?